tests/debug-tests/recv.py

```diff
@@ -119,16 +134,16 @@ def client(env, port, func):
         print(msg)
         assert rx > before_rx
 
-    cuda_obj_generator = cloudpickle.loads(func)
-    pure_cuda_obj = cuda_obj_generator()
+    # cuda_obj_generator = cloudpickle.loads(func)
+    # pure_cuda_obj = cuda_obj_generator()
 
-    from cudf.tests.utils import assert_eq
-    import cupy as cp
+    # from cudf.tests.utils import assert_eq
+    # import cupy as cp
 
-    if isinstance(rx_cuda_obj, cp.ndarray):
-        cp.testing.assert_allclose(rx_cuda_obj, pure_cuda_obj)
-    else:
-        assert_eq(rx_cuda_obj, pure_cuda_obj)
+    # if isinstance(rx_cuda_obj, cp.ndarray):
+    #     cp.testing.assert_allclose(rx_cuda_obj, pure_cuda_obj)
+    # else:
+    #     assert_eq(rx_cuda_obj, pure_cuda_obj)
```

Comment:
What's the reason for commenting this code?

Reply:
Any thoughts @quasiben? 🙂

Reply:
ooops.... thanks!